contracts/token/TribeSplitter.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.8.0;
+
+import "../pcv/utils/PCVSplitter.sol";
+
+/// @title TribeSplitter
+/// @notice a contract to split TRIBE held to multiple locations
+contract TribeSplitter is PCVSplitter {
```

Comment:
since there is no delegation of voting powers or anything, I guess you could even be more generic and make it an ERC20Splitter. We'll instanciate in on TRIBE through the constructor

Reply:
Changed :)